style.css

```diff
@@ -2107,6 +2107,11 @@ body.template-cover .entry-header {
 	font-family: 'Inter', -apple-system, BlinkMacSystemFont, 'Helvetica Neue', Helvetica, sans-serif;
 }
 
+.wp-block-pullquote.alignleft,
```

Comment:
Can you move this block of CSS to beneath the "Block: Pullquote" CSS subheading, at line 2291?